example_test.go

```diff
@@ -54,6 +56,36 @@ func Example() {
 	// {"msg":"Oh no!","level":"error","ts":0,"fields":{"user":"jane@test.com","visits":42}}
 }
 
+func Example_fileOutput() {
+	// Create a temporary file to output logs to.
+	f, err := ioutil.TempFile("", "log")
+	if err != nil {
+		panic("failed to create temporary file")
+	}
+	defer os.Remove(f.Name())
+
+	logger := zap.NewJSON(
+		// Write the logging output to the specified file instead of stdout.
+		// Any type implementing zap.WriteSyncer or zap.WriteFlusher can be used.
+		zap.Output(f),
+	)
+	// Stub the current time in tests.
+	logger.StubTime()
+
+	logger.Info("This is an info log.", zap.Int("foo", 42))
+
+	// Sync the file so logs are written to disk, and print the file contents.
```

Comment:
If we're including this in an example, can we call out that zap already syncs when logging at `PanicLevel` and `FatalLevel`?